libraries/AP_SerialLED/AP_SerialLED.cpp

```diff
@@ -30,20 +30,28 @@ AP_SerialLED::AP_SerialLED()
 // set number of LEDs per pin
 bool AP_SerialLED::set_num_LEDs(uint8_t chan, uint8_t num_leds)
 {
-    if (chan >= 1 && chan <= 16 && num_leds <= 32) {
+    if (chan >= 1 && chan <= 16 && num_leds <= 64) {
         return hal.rcout->set_neopixel_num_LEDs(chan-1, num_leds);
     }
     return false;
 }
 
 // set RGB value on mask of LEDs. chan is PWM output, 1..16
-void AP_SerialLED::set_RGB(uint8_t chan, uint32_t ledmask, uint8_t red, uint8_t green, uint8_t blue)
+void AP_SerialLED::set_RGB_mask(uint8_t chan, uint32_t ledmask, uint8_t red, uint8_t green, uint8_t blue)
 {
     if (chan >= 1 && chan <= 16) {
         hal.rcout->set_neopixel_rgb_data(chan-1, ledmask, red, green, blue);
     }
 }
 
+// set RGB value on LED number. LED number -1 is all LEDs. First LED is 0. chan is PWM output, 1..16
+void AP_SerialLED::set_RGB(uint8_t chan, int8_t led, uint8_t red, uint8_t green, uint8_t blue)
```

Comment:
How is this different from the mask version above? As far as I can tell the implementation actually looks identical, and if this is depending on a argument type difference it's way to close. Was the above mask one not calling `set_neopixel_rgb_data_mask` by accident?

Reply:
I've now removed the mask API. I think it is better to just have the one API, and the -1 for led meaning all LEDs covers most of the use case for mask